src/en/components/stepper/design.md

```diff
@@ -14,7 +14,7 @@ date: "git Last Modified"
 
 **The page H1**, or page title, is a call to action and states the goal of that particular step.
 
-<img class="b-sm b-gray p-400" src="/images/en/anatomy/gcds-stepper-anatomy.svg" alt=""/>
+<img class="b-sm b-gray p-400" src="/images/en/anatomy/gcds-stepper-anatomy.svg" alt="Page title taxonomy showing Page title in dark bolded text and above it Step1 of 4 in grey text smaller font. The taxonomy has these labels, Current step pointed at 1 of  Step 1 of 4. Stepper pointed tat Step 1 of 4. Total Steps pointed at 4, Page H1 pointed at the Page title."/>
```

Comment:
- Add space between "Step" and "1".
- Change "grey text smaller font" to "grey text and smaller font"?
- "Stepper pointed tat Step 1 of 4" -> "Stepper pointed at Step 1 of 4"